docs/guides/Troubleshooting/my-store-is-slow-or-unavailable.md

```diff
@@ -0,0 +1,111 @@
+
+Learn more about how to [set up DNS pointing to VTEX](https://help.vtex.com/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/12bQlMbJ68Ot0LIaO6Btkj#setting-up-dns-pointing).
+
+### Failure to generate SSL certificate
+
+If your store failed to go-live after you [notified DNS pointing](https://help.vtex.com/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/12bQlMbJ68Ot0LIaO6Btkj#notifying-the-pointing), it may be that there was a failure to generate your SSL certificate. Refer to this guide to learn how to identify this issue and what to do in case of problems after [DNS pointing notification](https://help.vtex.com/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/12bQlMbJ68Ot0LIaO6Btkj#notifying-the-pointing)
+
+### DNS provider settings
+
+It may be a good idea to double check your DNS provider settings.
+
+For example, if cloudflare is your DNS provider, their native proxy can interfere with SSL certificate, causing your store to go down. If this is your case, learn [how to disable the Cloudflare proxy](https://help.vtex.com/en/tutorial/disable-cloudflare-proxy--75QqsXAqR7NdkRc1GZPiXb).
+
+### Reverse proxy
+
+[Reverse proxy](https://help.vtex.com/en/tutorial/how-to-insert-a-reverse-proxy-in-front-of-vtex-services--4PFWsfRAKviNVPf1bYdiir) malfunction can cause store availability issues.
+
+When you use a reverse proxy, you give up the optimized edge service managed by VTEX and are responsible for the site's actual provisioning, which is tasked with the settings, monitoring and feature management such as header forwarding, cookies and cache management. If this is your case, follow up with your infrastructure team or provider to learn if this is the cause of your problem. VTEX does not recommend using reverse proxies.
+
+### Double check go-live procedures and timeline
+
+A store's [go-live](https://help.vtex.com/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/1iP90RcJvlrfQhnlxM54wo) is a delicate series of processes, some being time sensitive. For example, once you register your domain in VTEX, you have up to seven days to set up DNS pointing.
+
+See the [go-live guide](https://help.vtex.com/tracks/go-live-your-store--4Ns5FxIiksmjsdX2yOTduM/1iP90RcJvlrfQhnlxM54wo) to check whether there is anything you missed and how to fix it.
+
+## Store is slow
+
+The VTEX platform is designed to be multi-tenant and autoscalable, which means that it can handle the needs of several merchants simultaneously without impacting individual store performance. However, to ensure the health of your store, VTEX may occasionally throttle unintentionally abusive requests to the platform.
+
+If your store is experiencing slow performance, it is likely due to storefront code customizations or composable integrations. We recommend checking these elements to ensure they are not sending excessive requests to VTEX, which can cause a throttle issue or bottleneck for your store.
```

Comment:
```suggestion
To maintain the overall health of the platform, VTEX may occasionally throttle unintentionally abusive requests. Therefore, if you notice slow performance, it is recommended to review your storefront code customizations and composable integrations. Ensure that these elements are not generating excessive requests to VTEX, as this can lead to throttling issues or bottlenecks for your store. By optimizing your requests and ensuring they are within acceptable limits, you can improve the overall performance of your store while ensuring compliance with VTEX's request throttling mechanisms.
```

Reply:
Considerando que é troubleshooting, fiz essa sugestão com foco em "VTEX may occasionally throttle unintentionally abusive requests to the platform." e o pq disso.

Reply:
Boa! Implementei com uma adaptação pra manter a primeira frase. A ideia dessa parte é explicar reforçar o porque que o mais provável em caso de lentidão é que o problema seja do lado do cliente.